fitsnap3lib/scrapers/vasp_scraper.py

```diff
@@ -196,9 +197,19 @@ def scrape_configs(self):
             natoms = np.shape(self.data["Positions"])[0]
             pt.shared_arrays["number_of_atoms"].sliced_array[i] = natoms
             self.data["QMLattice"] = (self.data["Lattice"] * self.conversions["Lattice"]).T
-            del self.data["Lattice"]  # We will populate this with the lammps-normalized lattice.
+
+            # Populate with LAMMPS-normalized lattice
+            del self.data["Lattice"]  
+
+            # TODO Check whether "Label" container useful to keep around
```

Comment:
It will be, but will need significant code development that allows for re-grouping, this will get echoed into this container